manifests/m/Mariocube/BannerEditingTool/0.8/Mariocube.BannerEditingTool.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Mariocube.BannerEditingTool

```

Comment:
```suggestion
PackageIdentifier: Audacity.AudacityPortable
```

Reply:
Please update file name and path to match this change.

Reply:
This isn’t audicy but lots of Wii U tools

Reply:
> This isn’t audicy but lots of Wii U tools

Oh - I'm guessing that Audacity Portable might be one of the tools, possibly to edit Wii U mp3s.

Reply:
> > This isn’t audicy but lots of Wii U tools
> 
> Oh - I'm guessing that Audacity Portable might be one of the tools, possibly to edit Wii U mp3s.

Yes, there are lots of command line tools for Wii U